x-pack/plugins/secops/server/graphql/network/schema.gql.ts

```diff
@@ -13,6 +13,17 @@ export const networkSchema = gql`
     ip: String
   }
 
+  enum NetworkTopNFlowFields {
+    bytes
+    packets
+    ipCount
```

Comment:
Is `ipCount` stands for `unique destination IPs`? In this case I assume the we don't have to follow the display name when defining a variable?

Reply:
it is because it can be destination/source/client/server